### PR TITLE
Service sub

### PR DESCRIPTION
**Pull request by rvolosatovs**

### CODE REVIEW

File: unit/service/service.go [hunk 1]
```diff
@@ -136,75 +146,6 @@ func (sv *Unit) Active() unit.Activation {
 	case Stop, StopSigabrt, StopPost, StopSigkill, StopSigterm, FinalSigkill, FinalSigterm:
 		return unit.Deactivating
 	default:
```
Comment (bpowers) [inline unit/service/service.go]: what happens if you remove the `default: panic("unreachable")`?  Is the compiler smart enough to know those are the only possible cases?

**Comment by bpowers** [comment]
one comment/question, otherwise looks good to me :)

File: unit/service/service.go (hunk 1 above)
Comment (rvolosatovs) [inline unit/service/service.go]: I also hoped for that initially, but nope - it's not, so I had to add those
asd

Comment (bpowers) [inline unit/service/service.go]: bummer!

**Pull request merged**